vpr/src/route/multi_queue_priority_queue.cpp

```diff
@@ -37,7 +37,13 @@ static inline pq_index_t cast_RRNodeId_to_pq_index_t(RRNodeId node) {
     return static_cast<pq_index_t>(std::size_t(node));
 }
 
-void MultiQueuePriorityQueue::add_to_heap(const pq_prio_t& prio, const RRNodeId& node) {
+void MultiQueuePriorityQueue::add_to_heap(const pq_prio_t& prio, const RRNodeId& node, const RRNodeId& target_node) {
+    if (node == target_node) {
+#ifdef MQ_IO_ENABLE_CLEAR_FOR_POP
+        pq_.setMinPrio(new_total_cost);
```

Comment:
Shouldn't this be `prio`, not `new_total_cost`? How is this compiling, that variable was not declared? Was this tested?